android/cli/hooks/aar-transform.js

```diff
@@ -9,6 +9,7 @@
  */
 
 var AarTransformer = require('appc-aar-tools').AarTransformer;
+var appc = require('node-appc');
```

Comment:
No need to do this. `init()` is passed a 4th argument which is a `node-appc` instance.

```js
exports.init = function (logger, config, cli, appc) {
```

You'll have to pass `appc` into `registerHyperloopCompatibilityFixes()`.